hash/m20.xml

```diff
@@ -470,4 +475,630 @@ license:CC0-1.0
+            <dataarea name="flop" size="286720">
+                <rom name="giochi.img" size="286720" crc="e73b1d58" sha1="063bbea1ae3266a4abb31f1312219c2bf4a72e11"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="heimkehr1">
+        <description>Heimkehr 1</description>
+        <year>198?</year>
+        <publisher>Olivetti?</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="heimkehr1.img" size="286720" crc="abb14eab" sha1="6ca549a4e25621e83f5c309e6e08cadc732999fe"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="heimkehr">
+        <description>Heimkehr</description>
+        <year>198?</year>
+        <publisher>Olivetti?</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="heimkehr.img" size="286720" crc="e591888e" sha1="8a06418a943efc10110393c823cd3c82275cecf1"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="lena">
+        <description>Lena Graphic Demo</description>
+        <year>198?</year>
+        <publisher>D.Bucci, C.Groessler</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="lena.img" size="286720" crc="741589bc" sha1="53fb1517b26fdc083f29ec3394cff5af8969556e"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="lotto">
+        <description>Lottozahlen (6 aus 49)</description>
+        <year>1992</year>
+        <publisher>D.Steuer</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="lotto.img" size="286720" crc="381cc475" sha1="ff3bf8b45ab677b2507d73be7fb4a6096e3ff620"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="marilyn128k">
+        <description>Marilyn Monroe 128k</description>
+        <year>198?</year>
+        <publisher>Olivetti?</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="marilyn128k.img" size="286720" crc="ffd839ad" sha1="db6ff344c6f118992dd109b6700f7da74db14af4"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="marilyn">
+        <description>Marilyn Monroe</description>
+        <year>198?</year>
+        <publisher>Olivetti?</publisher>
+
+        <part name="flop1" interface="floppy_5_25">
+            <dataarea name="flop" size="286720">
+                <rom name="marilyn.img" size="286720" crc="b072cb66" sha1="841db31d0451a86f623915126f5b04fdf23c82df"/>
+            </dataarea>
+        </part>
+    </software>
+
+    <software name="mauerschiessen">
+        <description>Mauerschießen</description>
```

Comment:
Please keep short names terse.  Try to abbreviate them to eight characters if possible.